DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/src/CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ endif()
 set(EMULATOR_COMPILE_FLAGS "-Wall ${CONSTEXPR_STEPS} -fintelfpga ${AC_TYPES_FLAG} ${FILTER_SIZE_FLAG} ${PIXELS_PER_CYCLE_FLAG} ${MAX_COLS_FLAG} ${PIXEL_BITS_FLAG} -DFPGA_EMULATOR")
 set(EMULATOR_LINK_FLAGS "-fintelfpga ${AC_TYPES_FLAG} ${FILTER_SIZE_FLAG} ${PIXELS_PER_CYCLE_FLAG} ${MAX_COLS_FLAG} ${PIXEL_BITS_FLAG}")
 set(HARDWARE_COMPILE_FLAGS "-Wall ${CONSTEXPR_STEPS} -fintelfpga ${AC_TYPES_FLAG} ${FILTER_SIZE_FLAG} ${PIXELS_PER_CYCLE_FLAG} ${MAX_COLS_FLAG} ${PIXEL_BITS_FLAG}")
-set(REPORT_LINK_FLAGS "-fintelfpga -Xshardware ${PROFILE_FLAG} ${FLAT_COMPILE_FLAG} -Xsparallel=2 ${SEED_FLAG} -Xsboard=${FPGA_BOARD} ${FILTER_SIZE_FLAG} ${PIXELS_PER_CYCLE_FLAG} ${MAX_COLS_FLAG} ${PIXEL_BITS_FLAG} ${IP_MODE_FLAG} ${USER_HARDWARE_FLAGS}")
+set(REPORT_LINK_FLAGS "-fintelfpga -Xshardware ${PROFILE_FLAG} ${FLAT_COMPILE_FLAG} -Xsparallel=2 ${SEED_FLAG} -Xstarget=${FPGA_BOARD} ${FILTER_SIZE_FLAG} ${PIXELS_PER_CYCLE_FLAG} ${MAX_COLS_FLAG} ${PIXEL_BITS_FLAG} ${IP_MODE_FLAG} ${USER_HARDWARE_FLAGS}")
```

Comment:
I see you updated the README to use `FPGA_TARGET` instead of `FPGA_BOARD`.  Did you mean to update the CMakeLists.txt files to use `FPGA_TARGET` instead of `FPGA_BOARD` as well?

Reply:
This change uses -Xstarget, but I don't think that is an aoc option (at least as of now).  Did you mean -Xsboard=${FPGA_TARGET}?

Reply:
`-Xstarget` is available in aoc now.  We are moving from -Xsboard to -Xstarget since the latter is meaningful for both full-system and IP-authoring flows.

Reply:
That is good.  On the other hand, I think the CMakeLists.txt should use FPGA_TARGET, as that is what the README says

Reply:
Both the README and the CMakeLists.txt should be modified with this change

Reply:
This is done with my latest update.